cli/spinner_test.ts

```diff
@@ -132,11 +132,8 @@ Deno.test("Spinner constructor accepts each color", async () => {
 Deno.test("Spinner.color can set each color", async () => {
   const text = await spawnDeno(["cli/testdata/spinner_cases/set_color.ts"]);
 
-  const expectedStr = `${LINE_CLEAR}\u001b[30m⠋${COLOR_RESET} ` + // Black
-    `${LINE_CLEAR}\u001b[31m⠙${COLOR_RESET} ` + // Red
-    `${LINE_CLEAR}\u001b[32m⠹${COLOR_RESET} `; // Green
-
-  assertStringIncludes(text, expectedStr);
+  assertStringIncludes(text, `${LINE_CLEAR}\u001b[30m⠋${COLOR_RESET} `); // includes black spinner
```

Comment:
We could use `black()` from `std/fmt/colors`, right?

Reply:
Good point, but it looks like `std/fmt/colors` and `std/cli/spinner` use different codes (`0` vs `39`) for resetting the colors. I'm not sure it's fine to align them. Leaving this as is for now.

Reply:
That's strange. Can we create an issue for that so we can investigate further?